develop/migration_mt_query.rst

```diff
@@ -1,33 +1,101 @@
 .. _mt_query_migration:
 
-Prepare Application for Migration
-=================================
+Prepare Application for Citus
+=============================
 
-Once the distribution key is present on all appropriate tables, the application needs to include it in queries.
+Set up Development Citus Cluster
+--------------------------------
+
+When modifying the application to work with Citus, you'll need a database to test against. Follow the instructions to set up a :ref:`development` of your choice.
```

Comment:
This should probably say "development environment of your choice"

Reply:
That word is an internal section identifier that gets replaced by the title of the section in the user-visible output. So in this case it would say "to setup a Single-Machine Cluster of your choice."

An alternate form in RestructuredText is to say
```rst
:ref:`new name <section_id>`
```
to override the name.